src/Microsoft.AspNetCore.Mvc.Core/ApiExplorer/DefaultApiDescriptionProfile.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.AspNetCore.Mvc.ApiExplorer
+{
+    public abstract class DefaultApiDescriptionProfile : ApiDescriptionProfile
```

Comment:
Another layer of classes so that our built-in profiles can have their 'responses' mutated